packages/yelp_xsl.rb

```diff
@@ -0,0 +1,27 @@
+
+require 'package'
+
+class Yelp_xsl < Package
+  description 'yelp-xsl is a collection of programs and data files to help you build, maintain, and distribute documen$
```

Comment:
It looks like you missed the last part of the description and there is no closing quote at the end.